include/picotls.h

```diff
@@ -986,7 +986,11 @@ struct st_ptls_context_t {
      */
     struct {
         uint8_t bytes[PTLS_SHA256_DIGEST_SIZE];
+#ifdef _WINDOWS
+        unsigned int is_set : 1;
+#else
         uint8_t is_set : 1;
+#endif
```

Comment:
```suggestion
        unsigned is_set : 1;
```
We have single boolean bitfield with type `unsigned` elsewhere. Let's follow that.